tools/markdown-magic/src/templates/Client-Requirements-Template.md

```diff
@@ -0,0 +1,41 @@
+These are the platform requirements for the current version of Fluid Framework Client Packages.
```

Comment:
For reviewers: this becomes the new "source of truth" for the client requirements documentation.